app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -879,17 +879,18 @@ define([
             }
         ],
         'validate-emails': [
+            // TODO: logic is same in lib/web/mage/validation.js. Should we uniform?
```

Comment:
Please remove comment

Reply:
Originally, I added to refer to duplicated logic in `lib/web/mage/validation.js`. Could you tell me why we should remove it, is it related to `TODO:` notation?

Reply:
Originally, I added to refer to duplicated logic in `lib/web/mage/validation.js`. Could you tell me why we should remove it, is it related to `TODO:` notation?

Reply:
@tuanpht In fact we have multiple duplicated code for validation and I don't see any real useful information in this comment. It's just increase file size

Reply:
Ok, I thought Magento have feature "Minify JavaScript Files" which removes spaces, line breaks, and comments, but I haven't tried it yet

I had already mention the reference in commit message, so I will remove this comment

Reply:
I updated. Sorry, do I need to run test again?